README.md

```diff
@@ -0,0 +1,115 @@
+<p align="center">
+</p>
+
+SAM CLI (Beta)
+==============
+
+![Build
+Status](https://travis-ci.org/awslabs/aws-sam-cli.svg?branch=develop)
+![Apache-2.0](https://img.shields.io/npm/l/aws-sam-local.svg?maxAge=2592000)
+![Contributers](https://img.shields.io/github/contributors/awslabs/aws-sam-cli.svg?maxAge=2592000)
+![GitHub-release](https://img.shields.io/github/release/awslabs/aws-sam-cli.svg?maxAge=2592000)
+![PyPI version](https://badge.fury.io/py/aws-sam-cli.svg)
+
+[Join the SAM developers channel (\#samdev) on
+Slack](https://join.slack.com/t/awsdevelopers/shared_invite/enQtMzg3NTc5OTM2MzcxLTdjYTdhYWE3OTQyYTU4Njk1ZWY4Y2ZjYjBhMTUxNGYzNDg5MWQ1ZTc5MTRlOGY0OTI4NTdlZTMwNmI5YTgwOGM/)
+to collaborate with fellow community members and the AWS SAM team.
+
+`sam` is the AWS CLI tool for managing Serverless applications written
+with [AWS Serverless Application Model
+(SAM)](https://github.com/awslabs/serverless-application-model). SAM CLI
+can be used to test functions locally, start a local API Gateway from a
+SAM template, validate a SAM template, fetch logs, generate sample
+payloads for various event sources, and generate a SAM project in your
+favorite Lambda Runtime.
+
+Main features
+-------------
+
+-   Develop and test your Lambda functions locally with `sam local` and
+    Docker
+-   Invoke functions from known event sources such as Amazon S3, Amazon
+    DynamoDB, Amazon Kinesis Streams, etc.
+-   Start local API Gateway from a SAM template, and quickly iterate
+    over your functions with hot-reloading
+-   Validate SAM templates
+-   Get started with boilerplate Serverless Service in your chosen
+    Lambda Runtime `sam init`
+
+Get Started
+-----------
+
+Learn how to get started using the SAM CLI with these guides:
+
+-   [Installation](docs/installation.rst): Set up your macOS, Linux or
```

Comment:
Should we use this time to update these links to point to the aws docs we now have?

Reply:
definitely. the installation docs already do, but need to move out the rest too.

Reply:
I wonder if we should just collapse the installation docs into the readme. Not sure if the extra hop is valuable. Or just link directly to the landing page which has the different installations: https://aws.amazon.com/serverless/sam/

Or link them here: https://docs.aws.amazon.com/serverless-application-model/latest/developerguide/serverless-sam-cli-install.html but we need to update this one first to make pip an advanced install and default to installers first.

Reply:
yeah moved it to the landing page, this way the first options seen to install are the official installation options.